### ISSUE TITLE
masc/fem usages are separate lemmata: search list confusingly don't give this info

### ISSUE THREAD

**Issue by dwerning** [KELLIA/dictionary#63]
In the lexicon **masc. and fem.** usages of nouns are treated as seperate lemmata. Therefore, e.g. 
ⲣⲁⲙⲉ 'perch of Nile (Tilapia nilotica)', 
ⲣⲱⲙⲉ '1. man, human being, 2. ...' 
**appear "twice"** in the search result, with the same English translation (which does, other than German, not indicate the gender) -- however, without the critical info that the one is masc the other one fem (try search string "ⲣ.ⲙⲉ"). 
I **suggest to add the gender information** 
- in the result.cgi list
- in the "See also" list in entry.cgi

**Comment by amir-zeldes** [comment]
I believe the duplicates are in the XML data and are meant to support parallels with older Egyptian entries in TLA, where the genders have distinct entries - but maybe @phoenix-mossimo can say more about this?

**Comment by dwerning** [comment]
Yes, masc/fem instances are separate is in the XML data ... surely for a good reason. 
I suggest to **implement a display of the gender** (as in my comment above, https://github.com/KELLIA/dictionary/issues/63#issue-450646322).
Optionally in milestone 2.1.0.

**Comment by phoenix-mossimo** [comment]
Entry disambiguation according to gender is a requirement of TLA Egyptian lexicon with which Coptic will be matched. I agree, it would make sense to display gender in "result.cgi"  and "entry.cgi" to explain why some lemmata are shown twice. Adding a separate column is probably not the best option, because it will be empty with words other than nouns. Any ideas where this gender info should be displayed?

**Comment by amir-zeldes** [comment]
I'm also a bit uncertain about this, putting a gender on the entry may be confusing to users who get an entry page, say hmhal, and see it's masculine or feminine and think the noun only has this gender...

**Comment by dwerning** [comment]
I am not sure I understand the concern. Looking for hmhal would _always_ display all gender versions, right below each other, right? So there could be no misunderstanding, could it?

**Comment by amir-zeldes** [comment]
In string search you're right - the risk is really only if someone links to a specific entry, or does some automatic process that only keeps one of the two

**Comment by phoenix-mossimo** [comment]
> I'm also a bit uncertain about this, putting a gender on the entry may be confusing to users who get an entry page, say hmhal, and see it's masculine or feminine and think the noun only has this gender...

The gender for hmhal is shown in the entry. To show it also in the search results helps actually avoid the situation you describe - that someone cites / links hmhal (m) thinking that it is the only hmhal available.

**Comment by amir-zeldes** [comment]
I agree it could be nice to show it in search results (need to check space available, mobile view), but that's something I'd wait for to look into after release 2.0.0 - I'd really like to move on this and we keep adding features to this milestone.

My concern above was regarding a situation in which an entry is cited and a user clicks on a link to get directly to the entry "hmhal (M)", and then thinks that this word is always/only masculine. Maybe something to think about for the next release.

**Comment by dwerning** [comment]
@amir-zeldes The concern you describe might turn out not to be problematic: The direct link, e.g., to ϩⲙϩⲁⲗ (M) (.../entry.cgi?tla=C6507) lists ϩⲙϩⲁⲗ (F) under "**See also**:". Indeed displaying the gender will make the reader understand why (as currently) a seemingly twin ϩⲙϩⲁⲗ is listed under "See also".
Let's implement the display of gender in milestone 2.1.0.

**Comment by dwerning** [comment]
Users report that it is confusing that entries like
http://coptic-dictionary.org/entry.cgi?tla=C3272 (**masc**.)
vs.
http://coptic-dictionary.org/entry.cgi?tla=C3271 (**fem**.)
have the identical meaning "man (woman)".
Should this not be adapted to "man" (C3272) vs. "woman" (C3271)?

**Comment by dwerning** [comment]
The issue of "m/f-flexible" nouns turns out to be more tricky. For editoriyl-pragmatic reasons -- lemmatization of nouns without sg-article cannot always be identified as m _or_ f, we need a genus-neutral entry. 
For a list of ca. 200 cases, see https://github.com/thesaurus-linguae-aegyptiae/coptic-lexicon/issues/3

**Comment by pmissael** [comment]
Would it be possible to add both m. and f. under one entry?
For example:

Form | Dial. | Form ID | POS
ⲥⲁ | S | CF8432 | Subst. m.
ⲥⲁ | S | CF8430 | Subst. f.

This way, the user knows that the word can be masculine or feminine, and one entry for both is cited.
